dashboard/src/components/EventHeader.vue

```diff
@@ -10,24 +10,17 @@
       <div class="prose">
         <h1>{{ event.event_name }}</h1>
       </div>
-      <Badge
-        v-if="formExists && form.data"
-        :theme="form.data.is_published ? 'green' : 'gray'"
-        size="md"
-      >
-        <div class="font-medium">
-          <div v-if="form.data.is_published" class="flex items-center">
-            <span class="relative flex h-3 w-3 mr-2">
-              <span
-                class="animate-ping absolute inline-flex h-full w-full rounded-full bg-green-400 opacity-75"
-              ></span>
-              <span class="relative inline-flex rounded-full h-3 w-3 bg-green-500"></span>
-            </span>
-            <span>{{ form.data.doctype.split(' ').pop() }} Live</span>
-          </div>
-          <div v-else>
-            <span>{{ form.data.doctype.split(' ').pop() }} Unpublished</span>
-          </div>
+      <Badge v-if="formExists && form.data" :theme="getBadgeTheme()" size="md">
+        <div class="font-medium flex items-center">
+          <span v-if="shouldShowStatusDot()" class="relative flex h-3 w-3 mr-2">
+            <span
+              class="animate-ping absolute inline-flex h-full w-full rounded-full bg-green-400 opacity-75"
+            ></span>
+            <span class="relative inline-flex rounded-full h-3 w-3 bg-green-500"></span>
+          </span>
```

Comment:
I remember creating a component called LivePing.vue recently, we can refactor this to use that component

Reply:
Sure, will use that